plugins/aggregators/derivative/derivative.go

```diff
@@ -0,0 +1,209 @@
+package derivative
+
+import (
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/aggregators"
+)
+
+type Derivative struct {
+	Variable    string
+	Infix       string
+	MaxRollOver uint
```

Comment:
Please add the toml specifications here
```suggestion
	Variable    string   `toml:"variable"`
	Infix       string      `toml:"infix"`
	MaxRollOver uint `toml:"max_roll_over"`
```

Reply:
Resolved thanks!